src/api/app/websocket.ts

```diff
@@ -180,7 +175,17 @@ export class AppWebsocket implements AppClient {
     }
 
     const client = await WsClient.connect(options.url, options.wsClientOptions);
-    await client.authenticate({ token });
+    if (env?.APP_INTERFACE_TOKEN) {
+      await client.authenticate({ token: env.APP_INTERFACE_TOKEN });
```

Comment:
Maybe a comment here to note that the launcher must provide a `single_use = false` token?

Reply:
Hmm...good point. Do you see any problems with launcher providing `single_use = false` tokens?